homeassistant/components/zha/core/gateway.py

```diff
@@ -158,11 +157,10 @@ def __init__(self, hass, config, config_entry):
             self.application_controller.ieee
         )
         self._initialize_groups()
+        await self.async_load_devices()
```

Comment:
groups will need access to devices. can we move this above groups so that devices are loaded first. I can flip them later when I PR the group entities but they have always loaded first up until now anyway. Up to you...

Reply:
will do

Reply:
Why the tests did not catch it?

Reply:
It’s not happening yet. Will be in the group entities stuff mentioned above.